gpkit/geometric_program.py

```diff
@@ -174,8 +185,63 @@ def solve(self, solver=None, verbosity=1, *args, **kwargs):
                                  " in the 'result' attribute\n"
                                  "(model.program.result)"
                                  " and its raw output in 'solver_out'.")
-        else:
-            return result
+        
+        self.check_solution(result)
+        if verbosity > 1:
+            print ("solution checking took %.2g%% of solve time" %
```

Comment:
for some very simple problems I saw result packing taking around 30%, and solution checking around 2%. I doubt that's representative of real-scale problems though.